library/kotlin/src/io/envoyproxy/envoymobile/BUILD

```diff
@@ -52,15 +54,18 @@ envoy_mobile_kt_library(
         "ResponseTrailersBuilder.kt",
         "RetryPolicy.kt",
         "RetryPolicyMapper.kt",
+        "StatsClient.kt",
+        "StatsClientImpl.kt",
```

Comment:
Shouldn't these be in `stats/` which is captured below with globbing?

Reply:
I think these ideally live here, basically in the same level as `StreamClient`, as the interface `Engine.kt` exposes a `StatsClient` and a `StreamClient`.